subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -352,6 +296,119 @@ static void after_cen_offset_get(uint16_t conn_interval, uint32_t ticks_slot,
 #endif /* CONFIG_BT_CENTRAL */
 #endif /* CONFIG_BT_CONN */
 
+#if (defined(CONFIG_BT_CTLR_ADV_EXT) && defined(CONFIG_BT_BROADCASTER)) || \
+	defined(CONFIG_BT_CTLR_CENTRAL_ISO)
+static int group_free_slot_get(uint8_t user_id, uint32_t ticks_slot_abs,
+			       uint32_t *ticks_to_expire_prev,
+			       uint32_t *ticks_slot_prev,
+			       uint32_t *ticks_anchor)
+{
+	uint32_t remainder_prev;
+	uint8_t ticker_id;
+
+	ticker_id = after_match_slot_get(user_id, ticks_slot_abs,
+					 ticker_match_any_op_cb, ull_hdr_get_cb,
+					 ticks_anchor, ticks_to_expire_prev,
+					 &remainder_prev, ticks_slot_prev);
+	if (ticker_id != TICKER_NULL) {
+		uint32_t ticks_to_expire;
+		uint32_t ticks_slot;
+
+		ticks_to_expire = *ticks_to_expire_prev;
+		ticks_slot = *ticks_slot_prev;
+
+		if (false) {
+
+#if defined(CONFIG_BT_BROADCASTER)
```

Comment:
Can some of the `#if defined` here be changed into `IS_ENABLED`?

Reply:
There are defines used in `IN_RANGE` that are conditional hence, not able to use `IS_ENABLED` .